tern/analyze/docker/analyze.py

```diff
@@ -106,7 +106,12 @@ def analyze_subsequent_layers(image_obj, shell, master_list, redo, dfobj=None,
                               dfile_lock=False):
     # get packages for subsequent layers
     curr_layer = 1
+    #set current work_dir to none
```

Comment:
Small nit: redundant comment. Comments should be used when it's not totally clear what the code you're commenting does, but I think this line of code is pretty clear :)